maven-verify/action.yml

```diff
@@ -0,0 +1,28 @@
+name: Maven verify
+description: verify your maven jar
+runs:
+  using: "composite"
+  steps:
+    - id: get-build-parameters
+      run: |
+        branch_name=$(echo ${{ github.ref }} | cut -d "/" -f 3)
+        revision=$branch_name-$(git log -n1 --date=format-local:%G%m%d-%H%M --format='%ad' | tr -d " ")
+        sha=$(git log -n1 --format='%h'| tr -d " ")
+        build_number=${{ github.run_number }}
+        build_tag=build.$branch_name-$build_number
+        echo "BRANCH_NAME=$branch_name" >> $GITHUB_ENV
+        echo "REVISION=$revision" >> $GITHUB_ENV
+        echo "SHA=$sha" >> $GITHUB_ENV
+        echo "BUILD_NUMBER=$build_number" >> $GITHUB_ENV
+        echo "BUILD_TAG=$build_tag" >> $GITHUB_ENV
+      shell: bash
+
+    - id: maven-verify
+      run: |
+        mvn -B -Dbuild.tag=$BUILD_TAG verify \
+        -DimageRegistry=eu.gcr.io/bw-prod-analytics0 \
```

Comment:
This may needs to be configurable, given that we want to host image in AWS at some point

Reply:
You are right, I wanted to keep it simple for now, and do more configurations on a separate PR. But I am happy do it here if you want me to

Reply:
No need to address this now. I just wanted to point it out